exts/cesium.omniverse/cesium/omniverse/utils/utils.py

```diff
@@ -34,3 +42,50 @@ def str_is_empty_or_none(s: Optional[str]):
         return True
 
     return False
+
+
+def add_tileset_ion(name: str, tilesetId: int, token: str = ""):
```

Comment:
None of the tileset stuff belongs in this file. It should be moved to a different file. The utils file is for small UI related utilities.

Reply:
fair point, I've moved these into a `usdUtils` module

Reply:
Thanks for fixing this.